toolbox

```diff
@@ -661,6 +661,7 @@ create()
     monitor_host=""
     no_hosts=""
     toolbox_profile_bind=""
+    ulimit=""
```

Comment:
Nitpick: might be better to call it `ulimit_host` or such to steer clear of collisions with Bash's `ulimit` built-in.